core/frontend/src/tools/ViewTool.ts

```diff
@@ -3169,7 +3169,7 @@ export class FitViewTool extends ViewTool {
 }
 
 /** A tool that views a location on the background map from a satellite's perspective; the viewed location is derived from the position of the current camera's eye above the background map. Operates on the selected view.
- * @beta
+ * @public
```

Comment:
The overridden methods like `onDataButtonDown` and `onPostInstall` will appear in docs without documentation. They're not very interesting/useful vs the documented base class methods, so perhaps they should be marked as internal. @kabentley disagree?

Reply:
Any thoughts on this?

Reply:
Please mark them `@internal`. They are part of the implementation, not of the API.

Reply:
For all view tools?  I'll limit it to my view globe tools for this PR - but lots of other ones have them untagged.

Reply:
Yes - the ones affected by your PR.

Reply:
pushed this.

Reply:
resolve this conversation if looks good?